src/site/stages/build/drupal/graphql/bioPage.graphql.js

```diff
@@ -4,6 +4,8 @@
  */
 const entityElementsFromPages = require('./entityElementsForPages.graphql');
 
+const { cmsFeatureFlags } = global;
```

Comment:
Something wonky happened to allow this into `master`. I think there was a PR that removed a flag at the same time  another used a new flag. The one that removed it got rid of this line. The one that used a new flag didn't touch this, so we end up with a reference to `cmsFeatureFlags` without defining it.